### PR TITLE
Fix some broken manual links in the introduction

### PR DESCRIPTION
**Pull request by Announcement**
Bring back changes to fix broken manual changes including formatter, tester, linter and compiler.

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -16,14 +16,14 @@ It's built on V8, Rust, and Tokio.
 - Supports [TypeScript](./typescript.md) out of the box.
 - Ships a single executable (`deno`).
 - Provides built-in [development tooling](./tools.md) like a code formatter
-  ([`deno fmt`](./tools/formatter.md)), a linter
-  ([`deno lint`](./tools/linter.md)), a test runner
-  ([`deno test`](./testing.md)), and a
+  ([`deno fmt`](./manual/tools/formatter)), a linter
+  ([`deno lint`](.manual/tools/linter)), a test runner
+  ([`deno test`](./manual/testing)), and a
```
Comment (jsejcksn) [inline introduction.md:21]:
```suggestion
  ([`deno fmt`](./tools/formatter)), a linter
  ([`deno lint`](./tools/linter)), a test runner
  ([`deno test`](./testing)), and a
```

This document is already in the `/manual/` path.

File: introduction.md [hunk 2]
```diff
@@ -16,14 +16,14 @@ It's built on V8, Rust, and Tokio.
...
   [language server for your editor](./getting_started/setup_your_environment.md#using-an-editoride).
 - Has [a set of reviewed (audited) standard modules](https://deno.land/std/)
   that are guaranteed to work with Deno.
 - Can [bundle](./tools/bundler.md) scripts into a single JavaScript file or
-  [executable](./tools/compiler.md).
+  [executable](./manual/tools/compiler).
```
Comment (jsejcksn) [inline introduction.md:26]:
```suggestion
  [executable](./tools/compiler).
```

Same here.

**Review by jsejcksn — CHANGES_REQUESTED** [review]

**Comment by Announcement** [comment]
thank you @Announcement @jsejcksn, sorry for pushing a a second commit instead of accepting i'm not really the best at using git :P

**Comment by kt3k** [comment]
We landed https://github.com/denoland/dotland/pull/2177, and now `./manual/` redirects to `https://deno.land/manual@$VERSION/introduction` and these links are fixed now.

Thanks for your contribution anyways @Announcement

**Pull request closed without merging**